.gitattributes

```diff
@@ -0,0 +1,2 @@
+/install/kubernetes/tetragon/README.md linguist-generated
+/docs/content/en/docs/reference/helm-chart.md linguist-generated
```

Comment:
wow that's amazing, TIL! I explicitly excluded myself has a code owner from those files but I guess this is even better?